spinalcordtoolbox/deepseg/models.py

```diff
@@ -405,9 +405,10 @@
          'long_description': 'This segmentation model for spinal nerve rootlets segmentation uses a 3D U-Net '
                              'architecture, and was trained with the nnUNetV2 framework. It is a multiclass model, '
                              'outputting a single segmentation image containing 8 classes representing the C2-T1 '
-                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of 76 isotropic '
-                             'T2w and MP2RAGE images from healthy subjects from three datasets (spine-generic '
-                             'multi-subject, OpenNeuro ds004507, 7T MP2RAGE (private)).',
+                             'dorsal and ventral spinal cord nerve rootlets. Training data consisted of images from '
+                             'healthy subjects from three datasets: spine-generic multi-subject (3T T2w: n=21), '
+                             'OpenNeuro ds004507 (3T T2w, n=10), private data (7T MP2RAGE: n=15 (3 contrasts per '
```

Comment:
okay, @KaterinaKrejci231054 just slacked me that for OpenNeuro ds004507 she used  7 subjects / 10 images (the dataset contains multiple sessions, so subjects have >1 image). so to be precious, we should use something like:

```suggestion
                             'OpenNeuro ds004507 (3T T2w, 7 subjects, 10 images), private data (7T MP2RAGE: n=15 (3 contrasts per '
```